base/applications/network/netstat/netstat.c

```diff
@@ -609,10 +602,8 @@ GetIpHostName(BOOL Local, UINT IpAddr, CHAR Name[], INT NameLen)
 }
 
 /*
- *
  * Parse command line parameters and set any options
```

Comment:
```suggestion
 * Parse command line parameters and set any options.
```

Reply:
No, I didn't touch that line, and I don't want to add that.